tasks/run-buildpack-checksums-specs.yml

```diff
@@ -13,6 +13,10 @@ run:
     - |
       set -ex
       cd buildpack-checksums
+      if [ -n "${RUBYGEM_MIRROR}" ]; then
+        gem sources --clear-all --add ${RUBYGEM_MIRROR}
+      fi
+      gem install bundler --no-ri --no-rdoc
```

Comment:
Additionally, it'd be great to see the `--no-ri` and `--no-rdoc` flags used for the other instances of `gem install bundler` in the codebase.